Microsoft.Toolkit.Mvvm/Microsoft.Toolkit.Mvvm.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <Title>Windows Community Toolkit Mvvm .NET Standard</Title>
+    <Description>
+      This package includes Mvvm .NET Standard code only helpers such as:
+      - ObservableObject: a base class for objects implementing the INotifyPropertyChanged interface.
+      - ViewModelBase: a base class for view-models with built-in minimal dependency injection support.
```

Comment:
`ViewModelBase` is now called `ObservableRecipient` and the description should be updated to reflect that change and that it doesn't include any built-in dependency injection anymore.

Reply:
Done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3229/commits/ffcc807eec942aa868f61264e3aaf794543af3c5.